component-library/component-lib/src/lib/input/input.component.html

```diff
@@ -20,14 +20,21 @@
         <div class="input-content-area" [ngClass]="focusState === true ? 'focus' : ''">
           <!-- ngModel doesn't mind undefined values, apparently
           TODO: Should probably change the (keyup) to (onblur) -->
-          <input class="input-field" placeholder="{{ (config.placeholder || '') | translate }}"
+          <textarea *ngIf="(config.type === 'text' || !config.type) && !showPassword" class="input-text-field" [id]="config.id" 
```

Comment:
Why was this changed to textarea? This has other implications....

Reply:
So according to designs the text input field has multiple lines/rows so for that to work text-area seemed to be a good solution

Reply:
After checking with Tyh, the text-area field will be a separate story and this story only covers single line input, so removed the text-area and added back input for single-line non password inputs